src/components/Slider/Slider.test.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { render } from '@testing-library/react';
+
+import Slider from './Slider';
+
+describe('Slider', () => {
+  const defaultProps: React.ComponentProps<typeof Slider> = {
+    onChange: jest.fn(),
+    value: 50,
+    valueBoundaries: [0, 100],
+  };
+
+  test('default snapshot', () => {
+    const { asFragment } = render(<Slider {...defaultProps} />);
+    expect(asFragment).toMatchSnapshot();
```

Comment:
This snapshot doesn't contain anything except for a rendered function

Reply:
Good catch